src/inmanta/resources.py

```diff
@@ -292,7 +386,12 @@ def object_to_id(
 
     @classmethod
     def map_field(
-        cls, exporter: Optional["export.Exporter"], entity_name: str, field_name: str, model_object: "proxy.DynamicProxy"
+        cls,
+        exporter: Optional["export.Exporter"],
```

Comment:
I would expect the exporter object exposed to every export function to enabled the resource to register a reference even if it has no mutator (for plugins that have built-in, not transparent support)